gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -150,6 +152,12 @@ def construct_argparser(self) -> argparse.ArgumentParser:
             default=3,
             help="Number of times to retry cluster connection if it fails.",
         )
+        parser.add_argument(
+            "--run_single_trial",
```

Comment:
might be better to default to running a single trial while we are still in this development phase. the most frequent usage of gtsfm right now  (by us) involves running a single trial.